pkg/controller/labelidentity/label_group_index.go

```diff
@@ -272,6 +270,20 @@ func (i *LabelIdentityIndex) AddSelector(selector *types.GroupSelector, policyKe
 	return i.getMatchedLabelIdentityIDs(sItem)
 }
 
+// Dedup LabelIdentity IDs in-place.
+func DedupLabelIdentites(labelIdentityIDs []uint32) []uint32 {
```

Comment:
Is this function only for test purpose? It looks this function was moved out from `SetPolicySelectors`, but not called in `RemoveStalePolicySelectors`